badge-maker/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.0.0 [WIP]
+
+- Drop compatibility with Node 10
```

Comment:
Not strictly related to this PR but the next package release has to be v4 because we've dropped node10. I'm starting the changelog here so we don't lose sight of it.